src/components/customer/customerModel.js

```diff
@@ -80,7 +80,7 @@ const CustomerModel = sequelize.define(
       type: DataTypes.STRING,
       default:
         "https://shop.phuongdonghuyenbi.vn/wp-content/uploads/avatars/1510/default-avatar-bpthumb.png",
-      allowNull: true,
+      allowNull: false,
     },
   },
```

Comment:
Default customer's avatar is `null`. We generate avatar (FE side) for customer base on their name like **Skype** if they don't have avatar.

Reply:
Okay, Moreover, I'm considering when get customer from third party(they may have their avatar already.Thus, whether I should add the image_name column to this model then we will store their available avatar to our S3. However, we need Khoi to confirm the expected behavior.